### ISSUE TITLE
[COGNITION]: GIBCT - caution flags SHOULD NOT announce themselves dynamically to screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7982]
**Feedback framework**

- **⚠️Should** fix this issue 1-3 sprints post launch
- [508 defect 2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Description

The alert boxes all have `aria-live="polite"` regions added to them. NVDA and JAWS do not announce these because the content doesn't change, but VoiceOver does, and it is causing a lot of information to be read. VO users could get overwhelmed and have trouble associating alerts with schools. Let's remove those aria-live regions. Screenshots attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] VoiceOver does not announce the alerts until I cursor to them

## Environment

* MacOS Catalina
* Safari latest
* VoiceOver
* https://staging.va.gov/gi-bill-comparison-tool/search?category=school&name=Dodge+City
* https://staging.va.gov/gi-bill-comparison-tool/profile/14900416

## Steps to Recreate
1. Start VoiceOver on Safari (don't use Chrome)
1. Navigate to the search results page in question ( https://staging.va.gov/gi-bill-comparison-tool/search?category=school&name=Dodge+City )
1. Verify the alert box is not read aloud
1. Go back to the previous page using breadcrumbs
1. Initiate a new search for “Dodge City”
1. Verify the alert box is read aloud right away

## WCAG or Vendor Guidance (optional)

* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-14 at 7 51 11 AM](https://user-images.githubusercontent.com/934879/79227605-c7406f80-7e25-11ea-8aac-59d914b6f5a3.png)

---
![Screen Shot 2020-04-14 at 7 52 36 AM](https://user-images.githubusercontent.com/934879/79227673-df17f380-7e25-11ea-8c13-f339035b7974.png)

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12221

**Comment by sonntag-adam** [comment]
@1Copenut in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam This works as expected in Staging now. Re-tested in JAWS and VoiceOver, which were the two showing inconsistencies before. Moving to close.
